packages/experimental/Checkbox/src/useCheckbox.ts

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import {
+  useAsPressable,
+  useKeyCallback,
+  useOnPressWithFocus,
+  useViewCommandFocus,
+  useAsToggle,
+} from '@fluentui-react-native/interactive-hooks';
+import { CheckboxProps, CheckboxInfo, CheckboxState } from './Checkbox.types';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+
+/** Re-usable hook for FURN Checkbox.
```

Comment:
nit: Comment should start on line after the comment opener

Also I think the syntax for props/returns is `@param props` and `@returns` in JSDocs

Reply:
Thanks! Those are good callouts.